gm4_storage_drawers/data/storage_drawers/functions/all/check_storable.mcfunction

```diff
@@ -0,0 +1,109 @@
+execute if entity @s[nbt={Item:{id:"minecraft:golden_hoe", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:golden_pickaxe", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:golden_shovel", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:diamond_axe", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:diamond_hoe", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:diamond_pickaxe", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:diamond_shovel", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:shears", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:flint_and_steel", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:fishing_rod", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:wooden_sword", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:stone_sword", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:iron_sword", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:golden_sword", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:diamond_sword", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:bow", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:enchanted_book", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:tipped_arrow", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:shield", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:trident", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:crossbow", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:potion", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:splash_potion", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
+execute if entity @s[nbt={Item:{id:"minecraft:lingering_potion", Count:1b}}] run scoreboard players set @s gm4_sd_storable 1
```

Comment:
Put newlines at the end of each file. This applies to almost all function files.